VALIDATOR-URI-FIX-RUNBOOK.md

```
@@ -0,0 +1,102 @@
+Regenerate the exact bytes (source of truth):
+
+```bash
+cast calldata "initializeV2(string,string[],string[])" \
+ "https://metadata.fabrica.land/ethereum/0x5cbeb7a0df7ed85d82a472fd56d81ed550f3ea95/" \
+ "[ipfs://QmRH7d7TGJ3DymLSRimjnH5cNGHzYfcvUTUA1tM9gizFY8,ipfs://QmXRQx7wPxSwQDVVr1pTkiwvBHBUd1SYLbLgSn1Bvirqpc,ipfs://QmcgEJkgCwizvs6Tu12jCaNMGciRNtH8dLA2TRS3aYWStX,ipfs://Qmf6Aia6gJfRgGyGroYft3kjxsLUhJEhMYVKPKj2JwY41Z,ipfs://QmeRZqhU59Vpn4JQvggBVQ97uMfmS68utweUury8n5JLPR,ipfs://QmNxY3ooc4VXbW6ETd1wVAxvajZYWu81U95MmWJiNBQw14]" \
+ "[Fabrica US Trust v3.0,Fabrica US Trust v3.1,Fabrica US Trust v3.2,Fabrica US Trust v3.3,Fabrica US Trust v3.4,Fabrica US Trust v3.5]"
+```
+
+Precomputed value (selector `0x445b6ca2`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)